src/python/pants/util/fileutil.py

```diff
@@ -60,3 +60,12 @@ def do_copy():
       do_copy()
     else:
       raise
+
+
+def is_fileobj_definitely_closed(fileobj):
+  """Return whether fileobj points to a file object which is definitely closed.
```

Comment:
This method is a bit too context-specific to put in `util` I think.

Reply:
Agreed, it's not that pythonic either.

Reply:
Deleted!